README.md

```diff
@@ -39,17 +40,20 @@ x = torch.cat([t_, x_], dim=2)  # include time as a channel
 
 # Interpolate it
 coeffs = torchcde.natural_cubic_coeffs(x)
-X = torchcde.NaturalCubicSpline(coeffs)
+X = torchcde.CubicSpline(coeffs)
+
 
 # Create the Neural CDE system
 class F(torch.nn.Module):
     def __init__(self):
         super(F, self).__init__()
-        self.linear = torch.nn.Linear(hidden_channels, 
+        self.linear = torch.nn.Linear(hidden_channels,
                                       hidden_channels * input_channels)
+
     def forward(self, t, z):
         return self.linear(z).view(batch, hidden_channels, input_channels)
 
+
```

Comment:
To help reduce space in an already-large README, I'd prefer to avoid adding these two-line spaces.